pystemd/dbuslib.pyx

```diff
@@ -496,25 +496,28 @@ cdef class DBus:
       https://github.com/facebookincubator/pystemd/blob/master/examples/monitor.py
       """
 
-      cdef int r
+      IF LIBSYSTEMD_VERSION < 237:
+          raise DBusError(-1, "sd_bus_match_signal not available in libsystemd v%d" % LIBSYSTEMD_VERSION)
```

Comment:
Q: Whats the output of this if LIBSYSTEMD_VERSION is not defined? would LIBSYSTEMD_VERSION < 237 be true or false?